### PR TITLE
Remove ExperimentalUseFabric from ReactRootView

### PR DESCRIPTION
**Pull request by acoates-ms**
Given the scope of the planned changes for fabric we are going to go with a instance flag instead of a per ReactRootView property.

This cleans up some of the existing paper code to avoid confusion.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows/pull/10559)

### CODE REVIEW

File: vnext/Microsoft.ReactNative/Views/Image/ReactImage.h [hunk 1]
```diff
@@ -69,8 +69,6 @@ struct ReactImage : xaml::Controls::GridT<ReactImage> {
  private:
   xaml::Media::Stretch ResizeModeToStretch();
   xaml::Media::Stretch ResizeModeToStretch(winrt::Windows::Foundation::Size size);
-  winrt::Windows::Foundation::IAsyncOperation<winrt::Windows::Storage::Streams::IRandomAccessStream>
-  GetImageMemoryStreamAsync(ReactImageSource source);
```
Comment (asklar) [inline vnext/Microsoft.ReactNative/Views/Image/ReactImage.h:73]: these changes seem unrelated?

**Comment by chrisglein** [comment]
Fixes https://github.com/microsoft/react-native-windows/issues/10520?

**Comment by acoates-ms** [comment]
> Fixes #10520?

Yes it would.  Although I wasn't planning on backporting this by default.

File: vnext/Microsoft.ReactNative/Views/Image/ReactImage.h (hunk 1 above)
Comment (acoates-ms) [reply vnext/Microsoft.ReactNative/Views/Image/ReactImage.h]: Removed those changes.

File: vnext/Microsoft.ReactNative/Views/TouchEventHandler.cpp [hunk 2]
```diff
@@ -406,233 +399,26 @@ winrt::Microsoft::ReactNative::JSValue TouchEventHandler::GetPointerJson(const R
       {"altKey", pointer.altKey}};
 }
 
-#ifdef USE_FABRIC
-// This work should probably be delegated to the ComponentViews
-facebook::react::SharedEventEmitter EventEmitterForElement(
```
Comment (asklar) [inline vnext/Microsoft.ReactNative/Views/TouchEventHandler.cpp:411]: is the idea that all of this "if fabric" moves over to another file, or where is it going?

Comment (acoates-ms) [reply vnext/Microsoft.ReactNative/Views/TouchEventHandler.cpp:411]: Yeah, fabric will have an alternative version of this file - parts of this logic are in shared fabric code - others will be quite different from the xaml code.
